0015-estimator-interface.md

```diff
@@ -0,0 +1,457 @@
+# Extended Estimator Interface
+
+| **Status**        | **Proposed/Accepted/Deprecated** |
+|:------------------|:---------------------------------------------|
+| **RFC #**         | 0015                                         |
+| **Authors**       | Samantha Barron (samantha.v.barron@ibm.com)  |
+|                   | Lev Bishop (lsbishop@us.ibm.com)             |
+|                   | Ikko Hamamura (ikko.hamamura1@ibm.com)       |
+|                   | Ian Hincks (ian.hincks@ibm.com)              |
+|                   | Takashi Imamichi (imamichi@jp.ibm.com)       |
+|                   | Blake Johnson (blake.johnson@ibm.com)        |
+|                   | Chris Wood (cjwood@us.ibm.com)               |
+| **Deprecates**    | RFC that this RFC deprecates                 |
+| **Submitted**     | 2023-10-03                                   |
+| **Updated**       | YYYY-MM-DD                                   |
+
+
+## Summary <a name="summary"></a>
+
+The current `Estimator.run()` method requires that a user provide one circuit for every observable and set of parameter values that they wish to run.
+(This was not always the case via circuit indices, but the history will not be discussed here.)
+It is common, if not typical, for a user to want to estimate many observables corresponding to a single circuit. 
+Likewise, it is common, if not typical, for a user to want to supply multiple parameter value sets for the same circuit.
+This RFC proposes to fix these issues by changing the `Estimator.run()` signature in the following ways: 
+
+ 1. Take the transpose of the current signature; rather than accepting `circuits`, `parameter_values`, and `observables` as three different (iterable) arguments which are to be zipped together, instead localize the distinct tasks to be run via an iterable of triples `(circuit, parameter_values, observables)`.
+ 2. In combination with 1, extend `parameter_values` and `observables` to be array-valued, that is, add the ability to explicity and conveniently specify multiple parameter value sets and observables for a single circuit.
+
```

Comment:
This RFC only talks about Estimator, but I assume we want to do the same for Sampler, to keep the input format consistent if nothing else?

Reply:
Yes. We plan to put that in a separate RFC.